### ISSUE TITLE
False warning about unnecessary fat arrow

### ISSUE THREAD

**Issue by minib00m** [clutchski/coffeelint#565]
Got little piece of code:
```
        .on('delete', ((x)=> console.log "ahoy!"
        @_deleteAction(x)))
```
`@_deleteAction(x)` is idented as `.on(...)` function and this piece of code causes false message about unnecessary fat arrow.
But when I ident this piece of code nicely for example like this
```
        .on('delete',
          (x) =>
            console.log "ahoy!"
            @_deleteAction(x))
```
then message disappears

**Comment by swang** [comment]
Hi, this appears to generate two distinct outputs
```coffeescript
hello
.on('delete', ((x) => console.log "ahoy!"
@_deleteAction(x)))
```
outputs
```javascript
hello.on('delete', ((function(_this) {
  return function(x) {
    return console.log("ahoy!");
  };
})(this), this._deleteAction(x)));
```

and 
```coffeescript
hello
.on('delete',
  (x) =>
    console.log "ahoy!"
    @_deleteAction(x))
```
outputs
```javascript
hello.on('delete', (function(_this) {
  return function(x) {
    console.log("ahoy!");
    return _this._deleteAction(x);
  };
})(this));
```
